yarn.lock

```diff
@@ -1972,6 +1972,21 @@
     "@griffel/react" "^1.0.0"
     tslib "^2.1.0"
 
+"@fluentui/react-infobutton@9.0.0-beta.99":
```

Comment:
wondering if we should do the same for "@fluentui/react-alert": "9.0.0-beta.115", cc @ling1726